src/io/hdf/hdf_archive.h

```diff
@@ -89,15 +87,15 @@ class hdf_archive
   template<class Comm = Communicate*>
   hdf_archive(Comm c, bool request_pio = false) : file_id(is_closed), access_id(H5P_DEFAULT), xfer_plist(H5P_DEFAULT)
   {
-    H5Eget_auto2(H5E_DEFAULT, &err_func, &client_data);
-    H5Eset_auto2(H5E_DEFAULT, NULL, NULL);
+    if (!hdf_error_suppression::enabled)
+      app_warning() << "HDF5 library warnings and errors may appear in output.\n";
```

Comment:
I feel an exception is needed and we can avoid pulling OutputManager

Reply:
Good point